packages/framework/ai-collab/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": "../../../common/build/build-common/tsconfig.node16.json",
+	"include": ["src/**/*"],
+	"exclude": ["src/test/**/*"],
+	"compilerOptions": {
+		"rootDir": "./src",
+		"outDir": "./lib",
+		"exactOptionalPropertyTypes": false,
```

Comment:
What would it take to make this true? For presence, I just fixed some upstream cases because it was important to be able to have true. All of the new APIs should be crafted to work with `true`.

Reply:
It was here probably just due to copy paste, build runs fine without it.

Reply:
I fixed the issues in this package's types, but the build now complains about the types from Tree, which I noticed also [has it set to false](https://github.com/microsoft/FluidFramework/blob/main/packages/dds/tree/tsconfig.json#L15). Leaving it set to false due to that but added a comment explaining why.

Reply:
For reference:

![image](https://github.com/user-attachments/assets/d0f1d28b-60c4-48e4-b14d-30ab913ef8b9)

Reply:
Oops, not sure where I lost it. Added now.